DEV-3047 Missing CGACs/Subtiers Federal Hierarchy

dpb-bah opened the pull request:
**High level description:**
Adding missing cgacs/subtiers to Federal Hierarchy metrics.

**Technical details:**
(see above)

**Link to JIRA Ticket:**
[DEV-3047](https://federal-spending-transparency.atlassian.net/browse/DEV-3047)

The following are ALL required for the PR to be merged:
- [ ] Backend review completed
- [x] Tested locally

alburde1 commented:
I just ran this locally and among the missing CGACs (several of which I deleted from my table to test) are FRECs. We probably want to account for that no? For reference, here are the ones that were "missing" in my test: `"missing_cgacs": ["1601", "9566", "3301", "002", "1125", "1133", "1100", "1130", "003", "1602", "3302", "3300"]` (003 and 002 I deleted for testing purposes from my DB)

dpb-bah commented:
> I just ran this locally and among the missing CGACs (several of which I deleted from my table to test) are FRECs. We probably want to account for that no? For reference, here are the ones that were "missing" in my test: `"missing_cgacs": ["1601", "9566", "3301", "002", "1125", "1133", "1100", "1130", "003", "1602", "3302", "3300"]` (003 and 002 I deleted for testing purposes from my DB)

Good call, the agency codes I was testing with were of the cgac variety (mostly `097`) so I must've missed the FRECs. I'll add them to the `all_cgacs`.

alburde1 commented on dataactcore/scripts/load_federal_hierarchy.py:
```diff
@@ -143,7 +143,9 @@ def pull_offices(sess, filename, update_db, pull_all, updated_date_from, export_
-
+                        # store all the cgacs/subtiers loaded in from this run, to be filtered later
+                        metrics['missing_cgacs'].append(agency_code)
+                        metrics['missing_subtier_codes'].append(org.get('agencycode'))
```
Does this only happen to the lower tier agencies? Do we want to include the `top_sub_levels` in case they don't have any offices listed under them?

dpb-bah replied:
I would say keep out the `top_sub_levels` until we start processing them which will just remove lines 132-134.
